Makefile

```diff
@@ -194,6 +194,14 @@ build/undefined_syms.txt: undefined_syms.txt
 clean:
 	$(RM) -r $(ROM) $(ELF) build
 
+distclean: clean
+	$(RM) -r $(TEXTURE_BIN_DIRS)
```

Comment:
Shouldn't this be `ASSET_BIN_DIRS` instead of `TEXTURE_BIN_DIRS`?

Reply:
Looking at the rest of the makefile a bit closer, it seems like `TEXTURE_BIN_DIRS` is just a duplicate of `ASSET_BIN_DIRS`, so we should probably remove it and use `ASSET_BIN_DIRS` instead.
On a similar note, `TEXTURE_DIRS` seems useless and looks like it could be removed entirely.

Reply:
I can make a follow up PR after we merge this if you prefer (to clean up what I just mentioned)

Reply:
I don't have problems doing that cleanup myself, I can do it if you like.